### PR TITLE
neuron: devel version

### PR DESCRIPTION
**Pull request by ntkme**
This RP adds devel version for neuron.

### CODE REVIEW

File: neuron.rb [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+
+    depends_on "inter-views"
+  end
+
+  devel do
+    url "http://www.neuron.yale.edu/ftp/neuron/versions/alpha/nrn-7.4.rel-1324.tar.gz"
+    sha256 "cfc5270f9f06321dfaa3b55beb68dc4ec82cdfd747647bb1528e0ccc850de201"
+    version "7.4.rel-1324"
+
+    depends_on "inter-views" => :devel
```
Comment (dpo) [inline neuron.rb]: I don't think there's a way to do that.

**Comment by iml** [comment]
Would you mind splitting this up into two commits `inter-views 19 (devel)` and `neuron 7.4.rel-1324 (devel)`

**Comment by ntkme** [comment]
@iml Done.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**